src/FontPicker/FontPicker.test.js

```diff
@@ -22,7 +22,7 @@ const FontPickerWithState = ({ onChange, value: valueIn = "", ...rest }) => {
 /**
  * Tests
  */
-describe("Select", () => {
+describe("FontPicker", () => {
```

Comment:
@evoinea FYI this wasn't intended to be the component name. It's the description for the group of tests. In this case the tests are all to be with selection. It's fine to be FontPicker though :-)